examples/core/console_log/src/main.rs

```diff
@@ -49,6 +49,6 @@ struct Handler;
 
 impl ScriptHandler for Handler {
     fn on_message(&mut self, message: &str) {
-        println!("{message}");
+        println!("{}", &message);
```

Comment:
Is this a necessary change? I realize now that we don't actually state our minimum supported Rust version...

If you have strong opinions on this let me know but I feel like this feature is recent enough that we could make the Rust compiler version that supports it our MSRV.